recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -6,10 +6,12 @@ BUGTRACKER = "https://bugs.webkit.org/"
 LICENSE = "BSD-2-Clause & LGPL-2.0-or-later"
 LIC_FILES_CHKSUM = "file://Source/WebCore/LICENSE-LGPL-2.1;md5=a778a33ef338abbaf8b8a7c36b6eec80 "
 
+REQUIRED_DISTRO_FEATURES = "opengl"
+
 DEPENDS:append = " \
     libsoup \
-    bison-native gperf-native harfbuzz-native libxml2-native ccache-native ninja-native ruby-native cairo \
-    fontconfig freetype glib-2.0 harfbuzz icu jpeg pcre sqlite3 zlib libpng \
+    bison-native gperf-native harfbuzz-native libxml2-native ccache-native ninja-native ruby-native \
+    fontconfig freetype glib-2.0 harfbuzz icu jpeg pcre sqlite3 zlib libinput libpng libtasn1 \
```

Comment:
The `libinput` package should be only needed with `-DENABLE_WPE_PLATFORM=ON`, and the build recipe does not use that.